src/App.js

```diff
@@ -43,6 +44,7 @@ function App() {
         <Route path="/checkout" element={<CheckOut />} />
         <Route path="/order" element={<Orders />} />
```

Comment:
could add as a private route

Reply:
will do it,thanks

Reply:
user can go to checkout & Orders page only through Cart page, which is already a private route